aerolyzer/image_restriction_main.py

```diff
@@ -49,49 +49,50 @@ def program(fxn, data, exifData, pathname):
     return isVerified
 
 
-def check_image(filename):
-'''
-Purpose:        The purpose of this main function is to check all image restrictions
-                for use in the Aerolyzer app.
-Inputs:         string filename of image locally
-Outputs:        None
-Returns:        isVerified
-Assumptions:    N/A
-'''
+def check_image(filename,confPath="."):
+    '''
+    Purpose:        The purpose of this main function is to check all image restrictions
+                    for use in the Aerolyzer app.
+    Inputs:         string filename of image locally, optional string confPath
+    Outputs:        None
+    Returns:        isVerified
+    Assumptions:    N/A
+    '''
     #instantiate classes
-    fxn     = Fxn()
+    fxn     = Fxn(confPath)
     #Retrieve exif data
-    data    = Data(filename)
+    data    = Data(confPath)
     exifData = data.get_exif(filename, True, False)
     isVerified = program(fxn, data, exifData, filename)
     return isVerified
 
 
 def main():
-'''
-Purpose:        The purpose of this main function is to check all image restrictions and
-                produce the correct error message should one occur.
-Inputs:         string image (as sys.argv[1])
-Outputs:        None
-Returns:        N/A
-Assumptions:    N/A
-'''
+    '''
+    Purpose:        The purpose of this main function is to check all image restrictions and
+                    produce the correct error message should one occur.
+    Inputs:         string image (as sys.argv[1]), Config path(as sys.argv[2])
+    Outputs:        None
+    Returns:        N/A
+    Assumptions:    N/A
+    '''
     #instantiate classes
-    fxn     = Fxn()
+    
 
     #Retrieve exif data
     if(len(sys.argv) < 2):
```

Comment:
@wingarlo Where do these arguments come from?nothing is passed to main, and there is no prompt in main to get them.

Reply:
The arguments come from the command line if image_restriction_main is ran directly. I was making the assumption that the users operating from the app won't have the option to chose the config file location, but the people importing the aerolyzer library will. Though I did also make an optional argument in check_image (which is called by the app) so I can make it prompt app users as well if that would be preferred. 

I can also change the docstring so that it has these assumptions under "Assumptions"

Reply:
Ack'ed. Yes please update assumptions.